ui/src/app/app.module.ts

```diff
@@ -210,7 +212,8 @@ export function initializeApp(
     ReactiveFormsModule,
     CommonModule,
     SharedModule,
-    FormModule
+    FormModule,
```

Comment:
Can we please alias either FormModule (as OsmtFormModule) or FormsModule (as NgFormsModule)